CannibalisticZombies/Assets/Scripts/HealthUI.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class HealthUI : MonoBehaviour
+{
+    // HealthUI displays current health.
+    // Calling SetHealth() with new Damage amount update healthBar to that amount.
+    public Image healthBar;
```

Comment:
Setting a variable to public will make the code accessible from all other scripts, but we don't need to for certain variables. Instead of setting it to public, we can use the attribute `SerializeField` which exposes the variable into the inspector.

Ex:
`[SerializeField] private Image healthBar;`